app/javascript/crayons/Buttons/Button.jsx

```diff
@@ -73,7 +73,7 @@ ButtonNew.displayName = 'ButtonNew';
 
 ButtonNew.propTypes = {
   children: defaultChildrenPropTypes,
-  primary: PropTypes.bool,
+  variant: PropTypes.oneOf([undefined, 'primary', 'secondary']),
```

Comment:
Refactor (blocking): `variant` is optional, so `undefined` is not required in `PropTypes.oneOf(...)`

Reply:
Fixed!